src/tlo/methods/wasting.py

```diff
@@ -1,41 +1,1947 @@
+    for that child. Based on these measurements, the child can be diagnosed as well/MAM/(un)complicated SAM and
+    eventually scheduled for the appropriate treatment. If the child (attending or not) is still under 5 at the time of
+    the next growth monitoring, the next event is scheduled with age-dependent frequency.
+    """
+
+    def __init__(self, module, person_id):
+        super().__init__(module, person_id=person_id)
+        assert isinstance(module, Wasting)
+
+        self.TREATMENT_ID = "Undernutrition_GrowthMonitoring"
+        self.ACCEPTED_FACILITY_LEVEL = '1a'
+
+        rng = self.module.rng
+        p = self.module.parameters
+        person_age = self.sim.population.props.loc[self.target].age_exact_years
+        def get_attendance_prob(age):
+            if age < 1:
+                return p["growth_monitoring_attendance_prob_agecat"][0]
+            if age < 2:
+                return p["growth_monitoring_attendance_prob_agecat"][1]
+            else:
+                return p["growth_monitoring_attendance_prob_agecat"][2]
+        self.attendance = rng.random_sample() < get_attendance_prob(person_age)
+
+    @property
+    def EXPECTED_APPT_FOOTPRINT(self):
+        """Return the expected appointment footprint based on attendance at the HSI event."""
+        # TODO: for now assumed general attendance prob for <1 y old,
+        #  later may be excluded from here and be dealt with within epi module
+        # perform growth monitoring if attending
+        if self.attendance:
+            return self.make_appt_footprint({'Under5OPD': 1})
+        else:
+            return self.make_appt_footprint({})
+
+    def apply(self, person_id, squeeze_factor):
+        logger.debug(key='debug', data='This is HSI_Wasting_GrowthMonitoring')
+
+        df = self.sim.population.props
+        p = self.module.parameters
+
+        # TODO: Will they be monitored during the treatment? Can we assume, that after the treatment they will be
+        #  always properly checked (all measurements and oedema checked), or should be the assumed "treatment outcome"
+        #  be also based on equipment availability and probability of checking oedema? Maybe they should be sent for
+        #  after treatment monitoring, where the assumed "treatment outcome" will be determined and follow-up treatment
+        #  based on that? - The easiest way (currently coded) is assuming that after treatment all measurements are
+        #  done, hence correctly diagnosed. The growth monitoring is scheduled for them as usual, ie, for instance, for
+        #  a child 2-5 old, if they were sent for treatment via growth monitoring, they will be on tx for adequate nmb
+        #  of weeks, but next monitoring will be done in ~5 months after the treatment. - Or we could schedule for the
+        #  treated children a monitoring sooner after the treatment.
+        # no
```

Comment:
this looks like long note to self.  Is it covered in the methods document? Can we remove it? It seems odd to leave the thinking-allowed here.